azure-pipelines.yml

```diff
@@ -140,10 +169,9 @@ jobs:
         timeoutInMinutes: 120
 
 #
-# Checked test builds
+# Checked test builds - PullRequest
```

Comment:
Is this comment change accidental ? Below this line are PullRequest, Manual and Scheduled builds (not only PullRequest)

Reply:
I'll fix it in the upcoming change for checked jobs

Reply:
I will update